crates/diman_unit_system/src/codegen/generic_methods.rs

```diff
@@ -25,9 +25,13 @@ impl Defs {
     }
 
     pub fn generic_methods(&self) -> TokenStream {
-        self.storage_type_names()
-            .iter()
-            .map(|name| self.impl_method_for_generic_storage_type(name, &quote! { abs }))
-            .collect()
+        if cfg!(feature = "float_core") {
+            self.storage_type_names()
+                .iter()
+                .map(|name| self.impl_method_for_generic_storage_type(name, &quote! { abs }))
+                .collect()
+        } else {
+            quote! {}
+        }
```

Comment:
I'd be fine with that. We can also rework the `assert_is_close_{quantity,float}` methods in those cases .

Reply:
I'm now leaning toward not supporting a configuration without access to `FloatCore` since it's so light (see long top-level comment) I can't think of a tangible benefit.